onnxruntime/wasm/.gitignore

```diff
@@ -0,0 +1,3 @@
+build
```

Comment:
Do we need this file, .gitignore? Since we moved out cmake, we're not going to build wasm from this directory.

Reply:
will remove